nodebuilder/tests/swamp/config.go

```diff
@@ -4,20 +4,27 @@ import (
 	"time"
 
 	tn "github.com/tendermint/tendermint/config"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+
+	"github.com/celestiaorg/celestia-app/testutil/testnode"
 )
 
 // Components struct represents a set of pre-requisite attributes from the test scenario
 type Components struct {
-	CoreCfg *tn.Config
+	CoreCfg         *tn.Config
+	ConsensusParams *tmproto.ConsensusParams
+	SupressLogs     bool
 }
 
 // DefaultComponents creates a KvStore with a block retention of 200
-// In addition, the empty block interval is set to 200ms
+// In addition, the empty block interval is set to 50ms
 func DefaultComponents() *Components {
```

Comment:
optional nit: It's time to rename this to just Config, as there are no more dependencies/components. I minor thing to do that I can create an issue for